engine/runner/src/main/java/org/enso/runner/Utils.java

```diff
@@ -0,0 +1,69 @@
+package org.enso.runner;
+
+import java.io.File;
+import java.io.IOException;
+import scala.Tuple3;
+
+final class Utils {
+  private Utils() {}
+
+  /**
+   * Verifies path and project path.
+   *
+   * @param path file or project to execute
+   * @param projectPath project path or {@code null} if it hasn't been specified
+   * @return tuple with boolean, File to execute and path for project to use or {@code null} if
+   *     execution shall finish
+   */
+  static scala.Tuple3<Boolean, File, String> findFileAndProject(String path, String projectPath)
+      throws IOException {
```

Comment:
This seems to be +- replicating the logic of
https://github.com/enso-org/enso/blob/develop/engine/launcher/src/main/scala/org/enso/launcher/project/ProjectManager.scala#L34-L37